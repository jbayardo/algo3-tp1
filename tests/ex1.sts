			0,0,0,0,0,0,0,1,0,0,0,0,0,1,0,0,0,0,1,1,0,1,0,0,0,0,0,0,0,1,0,1,0,1,1,1,0,1,0,0,1,0,1,1,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,4,3,3,3,3,3,3,2,3,3,3,3,3,3,2,2,2,3,2,3,2,3,3,3,2,2,3,3,3,3,3,3,3,2,3,3,2,2,2,2,3,2,3,2,3,2,2,3,3,3,2,2,2,3,2,3,2,3,2,3,2,3,3,3,2,3,3,2,3,2,3,2,3,3,3,3,2,2,3,2,3,3,3,2,3,3,2,3,2,3,2,2,2,3,3,2,2,3,2,3,21,21,21,21,21,21,21,21,20,25,26,27,26,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,42,23,31,21,21,21,21,21,21,21,21,21,21,21,21,21,21,21,1065,1032,1049,1011,1045,1009,1045,1017,1010,1012,1085,1006,1010,1044,1008,1010,1170,1135,1012,1013,1066,1047,1009,1011,1022,1019,1013,1014,1006,1017,1060,1020,1026,1019,1012,1024,1005,1019,1015,1016,1060,1028,1022,1021,1028,1015,1008,1009,2109,1383,1403,1438,1423,1478,1438,1346,1377,1407,1423,1445,1241,1189,1224,1237,1196,1251,1211,1206,1191,1274,1213,1088,1085,1290,1143,1162,1274,1151,1113,1109,1022,1019,1021,1073,1017,1019,1018,1125,1162,1236,1220,1371,1230,1022,1005,1006,1013,1057,1010,1008,10401,10470,10533,10786,10401,10397,10153,10299,9891,10280,10239,10307,9647,9295,9401,9561,9657,9338,9970,10106,9667,9697,10250,9868,9158,9216,9448,10541,12499,9529,10182,10240,9272,9626,9159,10136,10479,9251,9777,10306,10022,9241,10533,10309,9201,9343,9334,10453,9573,9309,9828,10283,10330,11601,10311,10228,10276,9525,9576,10302,9502,9259,10330,10166,9273,9208,9508,10300,9378,9506,10310,10234,9788,9721,10461,10378,11406,12128,10937,10911,10757,10843,10716,10385,10565,10570,10263,11260,11168,10811,10082,9131,9860,10856,10613,10274,9289,10190,10396,11797,25313,23789,27108,25871,28801,25490,25828,26447,24537,24968,28552,27260,26905,26360,26609,26539,25763,26720,26517,24896,24864,26777,26732,26212,25289,25457,26695,24612,26871,26615,24553,29630,24777,25718,25951,24712,26565,26233,26233,25937,26639,25957,26297,29293,27335,29500,30654,26246,23592,23591,23362,23450,24288,25459,27083,25504,25311,23168,23370,23700,25835,25474,24749,26594,25159,23260,23398,23579,24075,26319,24804,28393,26090,23628,25439,26398,23446,23418,23460,23561,26078,25499,23357,23046,24543,23913,26494,24287,25814,25316,25133,24971,26291,25199,24690,25812,25903,26581,25598,23648
