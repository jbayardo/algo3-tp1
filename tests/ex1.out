10
60
812
9997
14662
257972
9607624
